packages/manager/apps/pci-rancher/src/pages/create/Create.page.tsx

```diff
@@ -53,8 +53,8 @@ export default function Create() {
     onError: (error) => {
       trackingPage(`${TRACKING_PATH}::${TrackingEvent.add}-error`);
       setHasRancherCreationError(true);
-      if (error?.response?.data?.message) {
-        setRancherCreationErrorMessage(error.response.data.message);
+      if (error?.response?.data) {
```

Comment:
The type of `error` declared in react-query documentation could never be nullable: 
`onError: (err: TError, variables: TVariables, context: TContext | undefined) => void`